helm/csi-vxflexos/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 appVersion: "2.1.0"
-kubeVersion: ">= 1.21.0 < 1.24.0"
+kubeVersion: ">= 1.21.0-0 < 1.24.0-0"
```

Comment:
un desired side effect can occur , k8s alpha or beta unstable version will also be supported with this fix
example these k8s versions should be not be supported but this check will allow it
v1.22.0-beta.2
v1.23.1-rc.0